pkg/operator/ceph/cluster/osd/config.go

```diff
@@ -37,11 +37,26 @@ const (
 key = %s
 `
 
-	// OSDs on PVC using a certain storage class need to do some tuning
+	// OSDs on PVC using a certain slow storage class need to do some tuning
 	osdRecoverySleep = "0.1"
 	osdSnapTrimSleep = "2"
 	osdDeleteSleep   = "2"
 
+	// OSDs on PVC using a certain fast storage class need to do some tuning
+	osdOpNumThreadsPerShardSSD         = "2"
```

Comment:
Instead of defining all these consts, how about we just define a map here for all the settings?

Reply:
Done.